docs/modding/suitmodding/addingblobs.md

```diff
@@ -0,0 +1,253 @@
+
+
+To continue on the previous adding/patching of parts article from before, you'll likely also want to;
+- modify the image in the menu to reflect the changes you've made 
+- change the suit's name in the menu
+- and even add in your own custom description for your suit. 
+
+I'll also be mentioning a potential issue that arises when you're trying to
+- add parts to your suit that don't come from a reward suit chunk (chunk0) which will have to be solved in the manifest. 
+
+
+
+:::note Documentation
+If you haven’t reviewed SMF’s documentation resources (the book icon on the left when you enable developer mode in SMF), consider skimming these then return here. I also made a little tutorial on adding parts with screenshots. Use it in combination with the previous article (because the code is unreadable). Here’s another excellent resource worth bookmarking for later:
+
+- [Hitman Resources Documentation](https://hitman-resources.netlify.app/documentation) - An advanced look at WOA’s engine systems.
+- [Adding Suit Parts guide](https://www.nexusmods.com/hitman3/articles/63) - A guide on adding parts with screenshots. 
+:::
+
+>First up, let's try changing just the image in the menu with your custom image. These are called **"blobs"**
+
+---
+
+## STEP 1: **LOCATE YOUR FILE**
+First, let's locate your suit within the **repository**, using either RPKG or GlacierKit.
+
+There are 2 methods for this part. I'll do **RPKG** first since it's a bit more easier to find and confirm your image directly. I still suggest you try the **GlacierKit** method too for reasons I will explain later on.
+
+
+### Method 1: Using RPKG
+
+1. Open **RPKG** 
+2. In **RPKG**, click on *Import* in the top left corner.
+3. Select *Import RPKGs folder* and locate your Hitman 3 runtimes folder (it usually looks like this: `D:\SteamLibrary\steamapps\common\HITMAN 3\Runtime`).
+4. Once the folder is imported, click on the big **REPO** tab button next to *Resource Overview*.
+5. Then, click *Load REPO from RPKGS folder*. Now you’ll see everything neatly ordered with a good overview of each category. This is the **Repository**.
+6. Expand the *Outfits* section below. Search for your outfit there. Or use the search bar on top.
+
+>Leave your **RPKG** program open after finding your suit
+
+### Method 2: Using GlacierKit
+
+1. Open **GlacierKit**
+2. Select the *game contents* tab on the left.
+3. - Copy the following **repository's** file hash  ```00204D1AFD76AB13``` and paste it on the search bar there and hit enter. 
+   - Or simply type ***pro.repo*** on the search bar and hit enter.
+Both methods should bring you to a single file called ***pro.repo***. 
+4. Open it by clicking on it (obviously) and then click on *Open in editor* in the new menu that popped up on the right. This is the **repository** but in **Glacierkit**
+
+:::tip
+You now know the exact name of the **repository** file. Next time you want to open the **repository** in **GlacierKit** just type ***pro.repo***.
```

Comment:
This tip is just restating step 3 but saying not to do part of step 3, you can just make step 3 "search pro.repo" instead of "copy this hash and then paste and hit enter but actually don't do that use pro.repo instead"